Formula/hdf5@1.8.rb

```diff
@@ -3,9 +3,8 @@ class Hdf5AT18 < Formula
   homepage "https://www.hdfgroup.org/HDF5"
   # NOTE: 1.8.23 is expected to be the last release for HDF5-1.8
   # (see: https://portal.hdfgroup.org/display/support/HDF5%201.8.22#HDF51.8.22-futureFutureofHDF5-1.8).
```

Comment:
Final release so may want to add deprecation and remove livecheck.
- https://portal.hdfgroup.org/display/support/HDF5%201.8.23#future
- https://github.com/HDFGroup/hdf5#release-schedule
- https://www.hdfgroup.org/2023/02/release-of-hdf5-1-8-23-newsletter-190/

Reply:
👍